lib/nightwatch-mocks-handler.js

```diff
@@ -33,8 +42,34 @@ _.extend(MocksHandler.prototype, {
    * or started manually using start/stop async callbacks in the format
    * { mockServer: { start: func({port}), stop: ... } }
    */
+
   before: function (options, callback) {
     var log = this.log;
+    var startServer = function (server, connectOptions, serverPlugin) {
+
+      //return server.register(serverPlugin).then(server.start); - This doesnt work
```

Comment:
@jhudson8 ,
I'll remove the comment. I was wondering if we could do something similar ( it doesnt work though).